grapple/types/streamfield.py

```diff
@@ -391,20 +390,17 @@ def resolve_image(self, info, **kwargs):
         }
     )
 
-    SnippetObjectType = SnippetTypes.get_object_type()
-    if SnippetObjectType is not None:
+    class SnippetChooserBlock(graphene.ObjectType):
```

Comment:
Previously we'd only expose the snippet chooser block if we had any registered snippets.

What are the pros/cons of always exposing it?

Reply:
It looks like it was not possible to construct `SnippetObjectType` if there were no snippets defined: https://github.com/torchbox/wagtail-grapple/blob/6cebc7be1fcca437fcadd454ced3c11758da77f8/grapple/types/snippets.py#L6-L10

I don't see any downsides to always exposing the snippet chooser block. I've tried this in a minimal project and StreamField queries still work.